src/test/java/spoon/test/compilation/CompilationTest.java

```diff
@@ -154,7 +146,7 @@ public void testNewInstance() throws Exception {
 		}
 
 	}
-
+/*
```

Comment:
I will give it a try, but I would prefer if you merge my changes first, because then I can create another PRs too, related to compiler this evening. Otherwise I can fix only test and cannot do more until it is merged. Because of conflicts ...